ci/requirements/minimum-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: climpred-minimum-tests
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - xesmf
+  - esmpy
+  - dask
+  - netcdf4
+  - ipython
```

Comment:
Is ipython needed? Isn’t this essentially jupyter?

Reply:
Yes, you use `from IPython.display import display_html` in `classes` for instance.